### PR TITLE
:tada: Create nodeJS server.

### PR DESCRIPTION
**Pull request by NomiMagnus**
Add nodeJS server.

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const cors = require("cors");
+const mongoose = require("mongoose")
+
+
+const corsOptions = require("./config/corsOptions");
+const conectDataBase = require("./config/dbConn")
+
+
+conectDataBase()
+const PORT = process.env.PORT || 7000
```
Comment (SaraOchayon) [inline server/server.js:14]: Is there any significance to this particular port?

**Review by SaraOchayon — COMMENTED** [review]
wow! well done!
Clear and beautiful writing!

File: server/server.js (hunk 1 above)
Comment (NomiMagnus) [reply server/server.js:14]: The port will be taken from ENV file, in end point- 7000.

Comment (SaraOchayon) [reply server/server.js:14]: ok:)

**Review by SaraOchayon — COMMENTED** [review]
upload to dev!

**Pull request merged**